app/bundles/ReportBundle/Crate/ReportDataResult.php

```diff
@@ -75,17 +147,90 @@ public function getType($column)
         return isset($this->types[$column]) ? $this->types[$column] : 'string';
     }
 
+    /**
+     * @return array<mixed>
+     */
+    public function getTotals()
+    {
+        return $this->totals;
+    }
+
+    /**
+     * @return array<mixed>
+     */
+    public function getGraphs()
+    {
+        return $this->graphs;
+    }
+
+    /**
+     * @return DateTime|null
+     */
+    public function getDateTo()
+    {
+        return $this->dateTo;
+    }
+
+    /**
+     * @return DateTime|null
+     */
+    public function getDateFrom()
```

Comment:
Also return types for all new methods
```suggestion
    public function getDateFrom(): ?DateTime
```